packages/calypso-build/jest.config.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
+	setupFilesAfterEnv: [ __dirname + '/jest/setup.js' ],
 	testEnvironment: 'node',
 	testMatch: [ '<rootDir>/**/test/*.js?(x)', '!**/.eslintrc.*' ],
-	transform: { '^.+\\.jsx?$': 'babel-jest' },
+	transform: {
+		'^.+\\.[jt]sx?$': 'babel-jest',
+		'\\.(gif|jpg|jpeg|png|svg|scss|sass|css)$': __dirname + '/jest/util/assets/transform.js',
```

Comment:
nit: can we use `path.join()` here -  does it make sense at all? is there any dis-/advantage?

Reply:
:thinking: Might make sense to use :+1: (Only for the `+` though, right? We shouldn't need it for all the hard-coded slashes since I think this should work regardless of what OS we're on.)

Reply:
If we're using `path.join`, which does seem like a good idea, we might as well let it build the whole path.

Reply:
Okay :sweat_smile: